lib/ConvoyLeader/src/LateralController.cpp

```diff
@@ -60,8 +60,9 @@
  * Public Methods
  *****************************************************************************/
 
-LateralController::LateralController()
+LateralController::LateralController() : ILateralController(), m_headingFinder()
 {
+    m_headingFinder.init();
```

Comment:
In this case you could do it in the constructor of the heading finder class instead of calling the init() method separately.